server.js

```diff
@@ -0,0 +1,15 @@
+const app = require("./src");
+const db = require("./src/databaseSetup");
+const { PORT } = require("./src/config");
+
+db.authenticate()
```

Comment:
I think, that we can simplify database initialisation and remove this file:
1. Move this db.authenticate() call into databaseSetup file.
2. In databaseSetup, wrap everything(creation of db instance and db.authenticate call) in function, which will receive `app` param, and export it
3. Call this function in main file index.js and pass app variable into
4. Remove this server.js

Reply:
/+/